pycbc/waveform/waveform.py

```diff
@@ -647,14 +647,13 @@ def get_td_waveform_from_fd(rwrap=0.2, **params):
     hc: pycbc.types.TimeSeries
         Cross polarization time series
     """
-
     # determine the duration to use
     full_duration = duration = get_waveform_filter_length_in_time(**params)
     nparams = params.copy()
 
     while full_duration < duration * 1.5:
         full_duration = get_waveform_filter_length_in_time(**nparams)
-        nparams['f_lower'] -= 1
+        nparams['f_lower'] *= 0.99
```

Comment:
Yeah, I think this was one of the things in my list of hacks last year!

Do you think multiplying by `0.99` might be a bit slow? If the `f_lower` is quite a bit off this might require a number of iterations? (Although if the call is fast it doesn't matter). Approving anyway, but will leave this to consider.